### PR TITLE
[Bugfix] Opened the password range: 24 chars

### PR DESCRIPTION
**Pull request by kcw-grunt**
## Issue
Users of version 1 had an issue with the password blocking the password making them unable to login to Litecoin Card .

## Approach
Open the range to a wide number `24`

### CODE REVIEW

**Comment by kcw-grunt** [comment]
cc: @zenithquinn this is a result of some user feedback.

File: loafwallet/RegistrationViewModel.swift [hunk 1]
```diff
@@ -172,7 +172,8 @@ class RegistrationViewModel: ObservableObject {
             return false
         }
         
-        guard (passwordString.count >= 6 && passwordString.count <= 10) else {
+        //MARK: -  Max Password 24 chars
+        guard (passwordString.count >= 6 && passwordString.count <= 24) else {
```
Comment (mosadialiou) [inline loafwallet/RegistrationViewModel.swift:176]: @kcw-grunt I don't know what was the reason to limit the password to 10 or even 24 chars. I do think we just have to check the min number of chars. If a user want he can create a 50 chars password length.

**Review by mosadialiou — CHANGES_REQUESTED** [review]

File: loafwallet/RegistrationViewModel.swift (hunk 1 above)
Comment (kcw-grunt) [reply loafwallet/RegistrationViewModel.swift:176]: yeah...the partner are not giving details....I
'll take that line out.

File: loafwallet/RegistrationViewModel.swift [hunk 2]
```diff
@@ -170,11 +170,7 @@ class RegistrationViewModel: ObservableObject {
         
         guard passwordString != "" else {
             return false
-        }
-        
-        guard (passwordString.count >= 6 && passwordString.count <= 10) else {
```
Comment (mosadialiou) [inline loafwallet/RegistrationViewModel.swift:175]: @kcw-grunt The maximum number of characters is not needed but I do think we should keep the minimum characters at 6. Thoughts?

Comment (kcw-grunt) [reply loafwallet/RegistrationViewModel.swift:175]: The Ternio team haven't provided a schema yet. Standby on this. I will update the PR soon.

Comment (mosadialiou) [reply loafwallet/RegistrationViewModel.swift:175]: ok

**Review by mosadialiou — APPROVED** [review]
LGTM

**Pull request merged**